libs/pages/clusters/src/lib/ui/page-clusters-create/step-general/step-general.tsx

```diff
@@ -69,7 +78,7 @@ export function StepGeneral(props: StepGeneralProps) {
       </div>
 
       <form onSubmit={onSubmit}>
-        <div className="mb-10 text-sm">
+        <div className="mb-10 hidden text-sm">
```

Comment:
I think you can remove it instead of hidden

Reply:
https://github.com/Qovery/console/pull/1697/commits/e5b8fdbb167bd9a90b8e14d4b566f9976329d4b2